prov/verbs/src/fi_verbs.c

```diff
@@ -705,6 +705,15 @@ static int vrb_read_params(void)
 		return -FI_EINVAL;
 	}
 
+	if (vrb_get_param_bool("use_dmabuf", "Enable dmabuf based memory "
+			       "registrations, if supported. Yes by default.",
+			       (int *)&vrb_gl_data.dmabuf_support)) {
+		VRB_WARN(FI_LOG_CORE, "Invalid value of use_dmabuf\n");
+		return -FI_EINVAL;
+	}
```

Comment:
This could end up trying to re-enable dmabuf support after we checked that support isn't available.  Maybe logic something like this?
```
if (dmabuf_support) {
    if (get_param_bool("disable_dmabuf", ...)
        ...
}
```
I don't really care if it's use vs disable.

Reply:
fixed.